smtpserver/server.go

```diff
@@ -2418,10 +2418,16 @@ func (c *conn) deliver(ctx context.Context, recvHdrFor func(string) string, msgW
 					m.MessageID = messageid
 					m.MessageHash = messagehash
 					acc.WithWLock(func() {
-						if hasSpace, err := acc.TidyRejectsMailbox(c.log, conf.RejectsMailbox); err != nil {
+						hasSpace := true
+						var err error = nil
+						if !conf.KeepRejects {
+							hasSpace, err = acc.TidyRejectsMailbox(c.log, conf.RejectsMailbox)
+						}
+						if err != nil {
 							log.Errorx("tidying rejects mailbox", err)
-						} else if hasSpace {
-							if err := acc.DeliverMailbox(log, conf.RejectsMailbox, m, dataFile, false); err != nil {
+						}
+						if hasSpace {
```

Comment:
i was hoping for:

```suggestion
						} else if hasSpace {
```

i don't work with github a lot. this feels overheadful for a these nits, sorry about that. if you know of a better way to do this, i'm interested.

Reply:
I don't work with Github a lot either so no idea I'm afraid. Done though!